reference/test.i

```diff
@@ -42,6 +42,8 @@ ATOMS
 << irradiation schedule >>
 FLUX 1100000000000000.0
 TIME 300.0 SECS
+FLUX 42.0
+TIME 200.0 SECS
```

Comment:
I think we're going to need a lot more test cases for this.
Unfortunately, the system tests are not available, but perhaps the [workshops](https://github.com/fispact/workshops) can provide some valuable test cases.